build.py

```diff
@@ -15,12 +15,12 @@ def change_secret_key(keygen):
     file_string = file_handle.read()
     file_handle.close()
     file_string = re.sub(
-        r'DJANGO_SECRET_KEY=.*', r'DJANGO_SECRET_KEY=' + keygen, file_string
+        r'DJANGO_SECRET_KEY=.*', fr'DJANGO_SECRET_KEY={keygen}', file_string
```

Comment:
Quick question, have you tested this line? :smile:

Reply:
Below is the output of `make runtests`. Kindly tell me if there are other tests that I have to run.
![Screenshot from 2020-09-22 13-04-42](https://user-images.githubusercontent.com/10897423/93854906-332bdb80-fcd4-11ea-8679-4b8265b220ea.png)